File: soap-ws/src/main/java/ws/Account.java
package ws;

import jakarta.xml.bind.annotation.XmlAccessType;
import jakarta.xml.bind.annotation.XmlAccessorType;
import jakarta.xml.bind.annotation.XmlRootElement;
import jakarta.xml.bind.annotation.XmlTransient;

import java.util.Date;

@XmlAccessorType(XmlAccessType.FIELD)
@XmlRootElement(name = "Account")
public class Account {

    private int id;
    private double balance;

    @XmlTransient // don't transport this field during serialisation
    private Date created_at;

    public Account() {
    }

    public Account(int id, double balance, Date created_at) {
        this.id = id;
        this.balance = balance;
        this.created_at = created_at;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public double getBalance() {
        return balance;
    }

    public void setBalance(double balance) {
        this.balance = balance;
    }

    public Date getCreated_at() {
        return created_at;
    }

    public void setCreated_at(Date created_at) {
        this.created_at = created_at;
    }
}

File: php-client/src/Type/Account.php
<?php

namespace Src\Type;

class Account
{
    /**
     * @var int
     */
    private $id;

    /**
     * @var float
     */
    private $balance;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     * @return Account
     */
    public function withId($id)
    {
        $new = clone $this;
        $new->id = $id;

        return $new;
    }

    /**
     * @return float
     */
    public function getBalance()
    {
        return $this->balance;
    }

    /**
     * @param float $balance
     * @return Account
     */
    public function withBalance($balance)
    {
        $new = clone $this;
        $new->balance = $balance;

        return $new;
    }
}

